backend/src/jobs/chips.ts

```diff
@@ -0,0 +1,59 @@
+import { NestFactory } from '@nestjs/core';
+import { NestExpressApplication } from '@nestjs/platform-express';
+import { AppModule } from '../app.module';
+import { datePST } from '../common/helpers';
+import { AppLogger } from '../logger/logger.service';
+import { datasource } from '../database/datasource';
+import { PersonnelService } from '../personnel/personnel.service';
+import { PersonnelEntity } from '../database/entities/personnel/personnel.entity';
+import { Status } from '../common/enums';
+import { isAfter } from 'date-fns';
+
+
+(async () => {
+  if (!datasource) {
+    await datasource.initialize();
+  }
+  const app = await NestFactory.create<NestExpressApplication>(AppModule, {
```

Comment:
I think if you're not using nunjucks template here you can use createNestContext, rather than NestFactory.create (I _think_ it's a bit more lightweight? Could be wrong - it is the usual recommended approach for jobs/lambdas etc)